radicle-cli/examples/rad-id.md

```diff
@@ -136,7 +136,7 @@ Our key was added to the `Rejected` set of `keys` and the `total`
 increased to `1`.
 
     Rejected
-    
+
     total: 1
     keys: [
       "z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi"
```

Comment:
Maybe this should be `"delegates": [ "did:key:<...>", ...]`

Reply:
Was thinking the same yes.